app/pyproject.toml

```diff
@@ -53,8 +53,10 @@ mozilla-nimbus-shared = "0.0.9"
 isort = "5.5.2"
 google-cloud-storage = "1.10.0"
 django-storages = "1.7.1"
+graphene-django = "^2.13.0"
 
 [tool.poetry.dev-dependencies]
+rope = "^0.17.0"
```

Comment:
Sneaking rope in? ;D  Does this integrate with vscode?  Or you just call it from the command line?

Reply:
I honestly have no idea what put that in. It's a mystery to me as well.

Reply:
Ok, I found out what happened. I guess I tried to use a VS Code refactor, and it said, "Mind if I install a Python refactor tool?", and I was like, "yea, whatever", and so VS Code installed it and made a record of it for poetry.

Reply:
Did we mean to revert the `rope` dev-dependency before merging this?

Reply:
I was not intending to, as its a useful dependency for VS Code. Due to the container setup its a major pain to always be re-installing this manually.